packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts

```diff
@@ -1389,19 +1345,15 @@ export const shortCodeMap = {
 	"0x8a7": "IdCompressor should not be operated normally when in a ghost session",
 	"0x8a8": "IdCompressor should not be operated normally when in a ghost session",
 	"0x8a9": "IdCompressor should not be operated normally when in a ghost session",
-	"0x8aa":
-		"MigrationShim should not be able to apply v1 ops as they shouldn't have been created locally.",
-	"0x8ab":
-		"SharedTreeShim should not be able to apply v1 ops as they shouldn't have been created locally.",
+	"0x8aa": "MigrationShim should not be able to apply v1 ops as they shouldn't have been created locally.",
+	"0x8ab": "SharedTreeShim should not be able to apply v1 ops as they shouldn't have been created locally.",
 	"0x8ac": "Could not find expected sub-directory.",
-	"0x8ad": '"key" property is missing from edit operation.',
-	"0x8ae": '"key" property in edit operation is misconfigured. Expected a string.',
-	"0x8af": '"subdirName" property is missing from "createSubDirectory" operation.',
-	"0x8b0":
-		'"subdirName" property in "createSubDirectory" operation is misconfigured. Expected a string.',
-	"0x8b1": '"subdirName" property is missing from "deleteSubDirectory" operation.',
-	"0x8b2":
-		'"subdirName" property in "deleteSubDirectory" operation is misconfigured. Expected a string.',
+	"0x8ad": "\"key\" property is missing from edit operation.",
+	"0x8ae": "\"key\" property in edit operation is misconfigured. Expected a string.",
+	"0x8af": "\"subdirName\" property is missing from \"createSubDirectory\" operation.",
```

Comment:
interesting we normalize these to " strings with escapes instead of ' strings, but should be fine.